test/IRGen/section_errors.swift

```diff
@@ -0,0 +1,42 @@
+// RUN: %target-swift-frontend -enable-experimental-feature SymbolLinkageMarkers -parse-as-library -emit-sil %s -o /dev/null -verify
+
+// REQUIRES: swift_in_compiler
```

Comment:
Maybe we're getting lucky because of `swift_in_compiler`, but any test with experimental features requires `asserts` as experimental features cannot be enabled in non-assert compilers.